themes/socialbase/templates/layout/region--header.html.twig

```diff
@@ -1,7 +1,38 @@
 {% if content %}
   <nav {{ attributes.addClass('navbar navbar-default navbar-fixed-top') }} role="banner">
     <div class="container container--navbar">
-      {{ content }}
+
+      {% if style == 'sky' %}
```

Comment:
I'm not really able to figure out why this change was made.

We can already control positions of all our navbar blocks using CSS. So no template changes are needed for this. Additionally we're actively hiding the site branding block and hardcoding our own logo back in this template. That feels as a step backwards. Finally a menu toggle is added here but it seems to be hidden when I look at the site with the Sky flavour enabled.